src/lib/common/statistics.cpp

```diff
@@ -159,6 +159,7 @@ int noOfIndividualSubscriptionRequest                    = -1;
 int noOfSimulatedNotifications                           = -1;
 int noOfBatchQueryRequest                                = -1;
 int noOfBatchUpdateRequest                               = -1;
+int noOfRegistrationRequest                              = -1;
```

Comment:
This involves a new item in the GET /stats (?) operation? Have it any impact in documentation?

Reply:
Good comment, made me realize that the stats aren't complete.
When a new Service is defined in `enum RequestType`, the function `statisticsUpdate`, that has a switch statement with `enum RequestType` starts to complain.
To fix the warning, a new case is added with the counter for statistics of the new Service.

However, the counter has to be used as well, in a service routine called `renderCounterStats`.

Reply:
Fixed in e03804e (including the new counter in GET /statistics).

About documentation:
No, I think nothing is needed. We don't go into this kind of detail in the documentation.

Reply:
I'm unsure about the impact in documentation... https://fiware-orion.readthedocs.io/en/master/admin/statistics/index.html section "Counter block" shows particular counters. However, this could be considered in the documentation as an example and not exahustive. Have a look to that piece of documentaiton and tell me what do you think, pls.

Reply:
OK, if that part is meant to be a complete list, then one more counter, *"registrations"*, must be added.

Reply:
Fixed in 53980e4

Reply:
Well... not sure if that part was mean to be a complete list. That's why I ask for opinion :)

However, modification in 3980e4 is fine, so ok.